Makefile

```diff
@@ -261,4 +255,16 @@ ci-verify:
 
 .NOTPARALLEL: ember-dist ember-dist-dev
 
--include packagespec.mk
+.PHONY: build
+build:
+	@echo "--> Building go"
+	@go build -v -tags "$(GO_TAGS)" -ldflags " -X $VERSION_PKG_PATH.Version=$(VAULT_VERSION) -X $VERSION_PKG_PATH.GitCommit=$(VAULT_COMMIT)" -o dist/
```

Comment:
We should probably maintain the current practice of using CGO_ENABLED=0

Reply:
I missed that.  I'm ok with only having it in the place you have it now, so long as we annotate the make target to make it clear that it's not intended for developer interactive use.